pkg/tests/pull/cases/samechartvariations/wantResults/base/charts/test-chart-release-1/kustomization.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- test-chart/templates/test-1.yaml
-- test-chart/templates/test-2.yaml
-- test-chart/templates/test-3.yaml
-- test-chart/templates/test-4.yaml
-- test-chart/templates/test-5.yaml
+- templates/test-1.yaml
+- templates/test-2.yaml
+- templates/test-3.yaml
+- templates/test-4.yaml
+- templates/test-5.yaml
```

Comment:
If the user did not specify a `releaseName`, then the chart templates ended up in `chart-name/templates/...`.  If they did, they would end up in `release-name/chart-name/templates/...` because the chart name prefix was not removed from the base path.